eos/effects/adaptivearmorhardener.py

```diff
@@ -13,6 +14,13 @@
 def handler(fit, module, context):
     damagePattern = fit.damagePattern
 
+    sFit = Fit.getInstance()
+    static_adaptive_behavior = sFit.serviceFittingOptions["useStaticAdaptiveArmorHardener"]
+
+    if (damagePattern.emAmount == damagePattern.thermalAmount == damagePattern.kineticAmount == damagePattern.explosiveAmount) and static_adaptive_behavior:
+        pyfalog.debug("Setting adaptivearmorhardener resists to uniform profile.")
+        return
```

Comment:
I think this is an excellent compromise for this particular problem from before. :)